### PR TITLE
Fix DevMenu call command for iOS

### LINKED ISSUE #978: Palette commands "Reload App" and "Show Dev Menu" don't work on iOS
These commands do work for android app, but do not for iOS

### PR DESCRIPTION
**Pull request by SounD120**
Fixes #978

### CODE REVIEW

File: src/debugger/appWorker.ts [hunk 1]
```diff
@@ -60,19 +60,48 @@ Object.defineProperty(process, "versions", {
     value: undefined
 });
 
+function getNativeModules() {
+    var NativeModules;
+    try {
+        // This method for old RN versions
+        NativeModules = global.require('NativeModules').DevMenu.reload();
```
Comment (SounD120) [inline src/debugger/appWorker.ts]:
```suggestion
        NativeModules = global.require('NativeModules');
```

File: src/debugger/appWorker.ts [hunk 2]
```diff
@@ -60,19 +60,48 @@ Object.defineProperty(process, "versions", {
     value: undefined
 });
 
+function getNativeModules() {
+    var NativeModules;
+    try {
+        // This method for old RN versions
```
Comment (ruslan-bikkinin) [inline src/debugger/appWorker.ts]:
```suggestion
        // This approach is for old RN versions
```

File: src/debugger/appWorker.ts [hunk 3]
```diff
@@ -60,19 +60,48 @@ Object.defineProperty(process, "versions", {
...
+        // ignore error and try another way for more recent RN versions
+        try {
+            var nativeModuleId;
+            var modules = global.__r.getModules();
+            var ids = Object.keys(modules);
+            for (var i = 0; i < ids.length; i++) {
+              if (modules[i].verboseName) {
+                 var packagePath = new String(modules[i].verboseName);
+                 if (packagePath.indexOf("react-native/Libraries/BatchedBridge/NativeModules.js") > 0) {
+                   nativeModuleId  = parseInt(i, 10);
```
Comment (ruslan-bikkinin) [inline src/debugger/appWorker.ts]:
```suggestion
                   nativeModuleId = parseInt(i, 10);
```

Comment (ruslan-bikkinin) [inline src/debugger/appWorker.ts]: Shouldn't it be `ids[i]` instead of `i`?

File: src/debugger/appWorker.ts [hunk 4]
```diff
@@ -60,19 +60,48 @@ Object.defineProperty(process, "versions", {
...
+                   break;
+                 }
+              }
+            }
+          if (nativeModuleId) {
+            NativeModules = global.__r(nativeModuleId)
```
Comment (ruslan-bikkinin) [inline src/debugger/appWorker.ts]:
```suggestion
            NativeModules = global.__r(nativeModuleId);
```

File: src/debugger/appWorker.ts [hunk 5]
```diff
@@ -60,19 +60,48 @@ Object.defineProperty(process, "versions", {
...
+          }
+        }
+        catch (err) {
+            // suppress errors
+        }
+    }
+    return NativeModules;
+}
+
 var vscodeHandlers = {
```
Comment (ruslan-bikkinin) [inline src/debugger/appWorker.ts]:
```suggestion
// Originally, this was made for iOS only
var vscodeHandlers = {
```

**Pull request merged**